client/src/views/devices/ExportRecoveryDevicePage.vue

```diff
@@ -0,0 +1,204 @@
+                    @click="downloadKey()"
+                  >
+                    {{ $t('RecoveryDevicePage.actions.download') }}
+                  </ion-button>
+                </div>
+                <div v-else>
+                  <ion-icon
+                    :icon="checkmarkCircle"
+                    class="checked"
+                  />
+                  {{ $t('RecoveryDevicePage.subtitles.fileDownloaded') }}
+                </div>
+              </div>
+              <a
+                ref="downloadLink"
+                v-show="false"
+              />
+              <div v-if="recoveryKeyDownloaded && recoveryFileDownloaded">
+                <ion-button
+                  class="return-btn button-outline"
+                  @click="routerNavigateTo('workspaces')"
+                >
+                  <ion-icon
+                    :icon="home"
+                    class="icon"
+                  />
+                  {{ $t('RecoveryDevicePage.actions.backToWorkspaces') }}
+                </ion-button>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </ion-content>
+  </ion-page>
+</template>
+
+<script setup lang="ts">
+import {
+  IonPage,
+  IonContent,
+  IonButton,
+} from '@ionic/vue';
+import MsInformativeText from '@/components/core/ms-text/MsInformativeText.vue';
+import { ref, inject } from 'vue';
+import { getPasswordFromUser } from '@/components/core/ms-modal/MsPasswordInputModal.vue';
+import { useI18n } from 'vue-i18n';
+import { exportRecoveryDevice, RecoveryDeviceErrorTag } from '@/parsec';
+import { NotificationManager, Notification, NotificationKey, NotificationLevel } from '@/services/notificationManager';
+import { routerNavigateTo } from '@/router';
+import { home, checkmarkCircle, fingerPrint } from 'ionicons/icons';
+
+const { t } = useI18n();
+
+enum ExportDevicePageState {
+  Start = 'start',
+  Download = 'download',
+}
+
+const state = ref(ExportDevicePageState.Start);
+let code = '';
+let file = '';
+const downloadLink = ref();
+const recoveryKeyDownloaded = ref(false);
+const recoveryFileDownloaded = ref(false);
+const notificationManager: NotificationManager = inject(NotificationKey)!;
+
+async function exportDevice(): Promise<void> {
+  const password = await getPasswordFromUser({
+    title: t('PasswordInputModal.passwordNeeded'),
+    subtitle: t('PasswordInputModal.enterPassword'),
+    inputLabel: t('PasswordInputModal.password'),
+    okButtonText: t('PasswordInputModal.validate'),
+  });
+  if (!password) {
+    return;
+  }
+  const result = await exportRecoveryDevice(password);
+  if (!result.ok) {
+    // TODO: showToast();
+    // result.error.tag === RecoveryDeviceErrorTag.WrongPassword
+    console.log('ERROR !');
+    return;
+  }
+  code = result.value.code;
+  file = result.value.file;
+  state.value = ExportDevicePageState.Download;
+}
+
+async function downloadKey(): Promise<void> {
+  download(code, 'Recovery_code.txt');
```

Comment:
Probably better to change the file name here, something closer to what V2 is doing https://github.com/Scille/parsec-cloud/blob/7d8cce2fad2573d04e52bb0971c3e0b449c3f051/parsec/core/local_device.py#L77
We may even translate the file name, have something like `"Parsec_Code_Récupération_${OrgID}"` in French, `"Parsec_Recovery_Code_${OrgID}"` in English (those are just example).

`OrgID` can be retrieved with `getClientInfo()`, in `parsec/login.ts`.